### PR TITLE
[full ci] Fix ci test failure in test 6-07

### PR DESCRIPTION
**Pull request by chengwang86**
Fix the ci test failure seen in builds 11898 and 11907

### CODE REVIEW

File: isos/bootstrap/bootstrap [hunk 1]
```diff
@@ -38,6 +38,9 @@ if mount -t ext4 /dev/disk/by-label/containerfs ${MOUNTPOINT}; then
     mount -t tmpfs -o size=64m tmpfs ${MOUNTPOINT}/.tether/
 
     # enable full system functionality in the container
+    ln -s lib64 ${MOUNTPOINT}/.tether/lib
+    mkdir -p ${MOUNTPOINT}/.tether/{lib64,usr/lib/iptables}
```
Comment (mhagen-vmware) [inline isos/bootstrap/bootstrap]: should be not ln to the dir after we mkdir it?

Comment (chengwang86) [inline isos/bootstrap/bootstrap]: @mhagen-vmware It looks like `ln` will force `mkdir` if the dir does not exist. For example, the following succeeded on my local machine:
```
mkdir aaa
ln -s aaa bbb
ls -al | grep bbb
```
output: `bbb -> aaa`

Comment (chengwang86) [inline isos/bootstrap/bootstrap]: @mhagen-vmware when you create symbolic links by `ln -s`, the target file does not have to exist beforehand.

Comment (mhagen-vmware) [inline isos/bootstrap/bootstrap]: sure but it just increases readability maybe

**Pull request merged**